cmd/check.go

```diff
@@ -21,6 +21,14 @@ var checkFileCmd = &cobra.Command{
 		path := args[1]
 
 		runtimeData := getInitialRuntimeData(cmd)
+		defer func() {
+			if err := runtimeData.DeleteRuntimeTemp(); err != nil {
+				fmt.Printf("Failed to remove temporary VRCT files"+
+					"\n You should remove them manually in /tmp or reboot your device \n%s", err.Error())
+			}
+			os.Exit(1)
```

Comment:
Shouldn't this `os.Exit(1)` call be inside `if` block?

Reply:
Fixed